packages/records/lib/vitest.d.ts

```diff
@@ -0,0 +1,14 @@
+export * from 'vitest';
+
+/* eslint-disable @typescript-eslint/no-empty-interface */
+export interface CustomMatchers<TR = unknown> {
+    toBeIsoDate: () => TR;
+    toBeIsoDateTimezone: () => TR;
+    toBeUUID: () => TR;
+}
+
+declare module 'vitest' {
+    export interface Assertion<T = any> extends CustomMatchers<T> {}
+    export interface AsymmetricMatchersContaining extends CustomMatchers {}
+    export interface ExpectStatic<T = any> extends CustomMatchers<T> {}
+}
```

Comment:
Tried to use the one from `shared` but couldn't make it work. @bodinsamuel I saw you had the same issue with the `logs` package.

Reply:
yeah painful, I don't understand why